tidb-cloud/dev-guide-wordpress.md

```diff
@@ -96,6 +97,12 @@ Configure the WordPress database connection to TiDB Serverless.
 
 2. Set up your WordPress site by visiting [localhost](http://localhost/) if you start the container on your local machine or `http://<your_instance_ip>` if the WordPress is running on a remote machine.
 
+### Step 5: Confirm the database connection
+
+1. Close the connection dialog for your cluster on the TiDB Cloud website, and open the **Chat2Query** terminal.
```

Comment:
"console" is a more commonly used word when we refer to <https://tidbcloud.com/>.
```suggestion
1. Close the connection dialog for your cluster on the TiDB Cloud console, and open the **Chat2Query** terminal.
```